autonomy/cli/deploy.py

```diff
@@ -67,7 +67,7 @@
 def _validate_packages_path(path: Optional[Path] = None) -> Path:
     """Find packages dir for development mode."""
     path = Path(path or Path.cwd()).resolve()
-    while path != path.root:
+    while path != Path(path.root):
```

Comment:
i think it's better to get rid of while and make a list of all parent paths with for cycle by splitting path on components
```
In [5]: list(p.parents)
Out[5]: [PosixPath('/home/solarw'), PosixPath('/home'), PosixPath('/')]
```